bedrock/firefox/redirects.py

```diff
@@ -658,6 +658,8 @@ def firefox_channel(*args, **kwargs):
     redirect(r'^firefox/unfuck/?$', 'firefox.campaign.unfck.index'),
     redirect(r'^firefox/love/?$', 'firefox.campaign.unfck.index'),
     redirect(r'^firefox/liebe/?$', 'firefox.campaign.unfck.index'),
+    redirect(r'^firefox/rendonslenetplusnet/?$', 'firefox.campaign.unfck.index'),
```

Comment:
This forwards me to the en-US page, anything easy we can do about that?

Reply:
It should respect language preference same as the rest of the redirects